go/obscuronode/enclave/enclave.go

```diff
@@ -155,7 +163,7 @@ func (e *enclaveImpl) IngestBlocks(blocks []*types.Block) []nodecommon.BlockSubm
 func (e *enclaveImpl) SubmitBlock(block types.Block) nodecommon.BlockSubmissionResponse {
 	_, foundBlock := e.storage.FetchBlock(block.Hash())
 	if foundBlock {
-		return nodecommon.BlockSubmissionResponse{IngestedBlock: false}
+		return nodecommon.BlockSubmissionResponse{IngestedBlock: false, BlockNotIngestedCause: "Block already ingested."}
```

Comment:
I think this is fine.
Something to keep in mind for the future is that anything you return in this response is info that goes out of the enclave to the host, which can use it to probe for stuff.

Reply:
Why do we need the BlockNotIngestedCause sent upwards ? Should we just log ?

Reply:
I chose to do this because in the Docker case, the logs would be on the container side, and not accessible to the host process.

Reply:
Having logs in the docker container side is fine. I wouldn't implement custom logic for passing logs around.

Further down the road we can implement logic to aggregate logs from all the different running containers.

Reply:
I say we leave it for now, since it's there anyway. And we'll revisit later this entire object.